client/components/domains/domain-search-results/index.jsx

```diff
@@ -84,7 +92,9 @@ var DomainSearchResults = React.createClass( {
 					'%(cost)s.{{/small}}', { args: { domain, cost: this.props.products.domain_map.cost_display }, components } );
 			}
 
-			const domainUnavailableMessage = this.translate( '%(domain)s is taken.', { args: { domain } } );
+			const domainUnavailableMessage = this.noAvailabilityInfoForDomain()
+				? this.translate( '%(domain)s might be taken.', { args: { domain } } )
```

Comment:
I think we should say something along the lines of "not offered on WPCOM" instead of "might be taken".